cr.sh

```diff
@@ -177,6 +177,17 @@ lookup_latest_tag() {
     fi
 }
 
+filter_changed_charts() {
+    while read chart; do
+        local file="${chart}/Chart.yaml"
+        if [[ -f "${file}" ]]; then
+            echo ${chart}
+        else
+           echo "Warning: ${file} is missing, assuming that '${chart}' is not a helm chart. Skipping." 1>&2
```

Comment:
Remove braces to match existing style.
```suggestion
        local file="$chart/Chart.yaml"
        if [[ -f "$file" ]]; then
            echo "$chart"
        else
           echo "Warning: $file is missing, assuming that '$chart' is not a Helm chart. Skipping." 1>&2
```